src/documentation/pipelines/design-your-pipeline.md

```diff
@@ -70,14 +70,14 @@ PCollection<String> bCollection = dbRowCollection.apply("bTrans", ParDo.of(new D
 }));
 ```
 
-### A single transform that uses side outputs
+### A single transform that uses additional outputs
 
-Another way to branch a pipeline is to have a **single** transform output to multiple `PCollection`s by using [side outputs]({{ site.baseurl }}/documentation/programming-guide/#transforms-sideio). Transforms that use side outputs, process each element of the input once, and allow you to output to zero or more `PCollection`s.
+Another way to branch a pipeline is to have a **single** transform output to multiple `PCollection`s by using [additional outputs]({{ site.baseurl }}/documentation/programming-guide/#transforms-outputs). Transforms that use additional outputs process each element of the input once, and allow you to output to zero or more `PCollection`s.
```

Comment:
"tagged outputs"

Transforms that use additional outputs -> Transforms that produce more than one output?